<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterMain</class>
 <widget class="QMainWindow" name="RegisterMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titleLable">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>账户注册</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="passWordLable">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="checkPassWordLable">
          <property name="text">
           <string>确认密码</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="emailLable">
          <property name="text">
           <string>邮箱</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="passWordLineEdit">
          <property name="styleSheet">
           <string notr="true">border: 2px solid #bdc3c7;
color: #34495e;
font-family: &quot;Lato&quot;, Helvetica, Arial, sans-serif;
font-size: 15px;
border-radius: 6px;
box-shadow: none;
background-color: #fff;
background-clip: padding-box;
display: block;
width: 100%;
text-align: left;
max-height: 50px</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="checkPassWordLineEdit">
          <property name="styleSheet">
           <string notr="true">border: 2px solid #bdc3c7;
color: #34495e;
font-family: &quot;Lato&quot;, Helvetica, Arial, sans-serif;
font-size: 15px;
border-radius: 6px;
box-shadow: none;
background-color: #fff;
background-clip: padding-box;
display: block;
width: 100%;
text-align: left;
max-height: 50px</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="emailLineEdit">
          <property name="styleSheet">
           <string notr="true">border: 2px solid #bdc3c7;
color: #34495e;
font-family: &quot;Lato&quot;, Helvetica, Arial, sans-serif;
font-size: 15px;
border-radius: 6px;
box-shadow: none;
background-color: #fff;
background-clip: padding-box;
display: block;
width: 100%;
text-align: left;
max-height: 50px</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="statusLable">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nickNameLineEdit">
          <property name="styleSheet">
           <string notr="true">border: 2px solid #bdc3c7;
color: #34495e;
font-family: &quot;Lato&quot;, Helvetica, Arial, sans-serif;
font-size: 15px;
border-radius: 6px;
box-shadow: none;
background-color: #fff;
background-clip: padding-box;
display: block;
width: 100%;
text-align: left;
max-height: 50px</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nickNameLable">
          <property name="text">
           <string>昵称</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="userNameLable">
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="userNameLineEdit">
          <property name="styleSheet">
           <string notr="true">border: 2px solid #bdc3c7;
color: #34495e;
font-family: &quot;Lato&quot;, Helvetica, Arial, sans-serif;
font-size: 15px;
border-radius: 6px;
box-shadow: none;
background-color: #fff;
background-clip: padding-box;
display: block;
width: 100%;
text-align: left;
max-height: 50px</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>60</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="maleRadio">
             <property name="text">
              <string>男</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="famaleRadio">
             <property name="text">
              <string>女</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="sexCheckBox">
          <property name="text">
           <string>性别</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="OkBtn">
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: #48c9b0;
border-color: #48c9b0;
font-size: 17px;
border-radius: 6px;
font-weight: normal;
border: none;
border: 2px solid #bdc3c7;
display: block;
width: 100%;
float: left;
margin: 2px 0 2px;
font: 9pt &quot;黑体&quot;;
font-size: 11px;
padding-top: 5px;
padding-bottom: 5px;</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CancelBtn">
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: #48c9b0;
border-color: #48c9b0;
font-size: 17px;
border-radius: 6px;
font-weight: normal;
border: none;
border: 2px solid #bdc3c7;
display: block;
width: 100%;
float: left;
margin: 2px 0 2px;
font: 9pt &quot;黑体&quot;;
font-size: 11px;
padding-top: 5px;
padding-bottom: 5px;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
